.travis.yml

```diff
@@ -4,4 +4,10 @@ rvm:
   - 2.2.2
   - 1.9.3
   - jruby
+gemfile:
+  - gemfiles/Gemfile.rails-3.2
```

Comment:
These should live in `spec/gemfiles` which would also require `bundler_args` to work with caching. See for example https://github.com/jayroh/thredded/blob/master/.travis.yml